WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -178,14 +178,23 @@ class GutenbergSettings {
         }
 
         let rollout = GutenbergOnboardingRollout()
-        let hasLaunchedGutenbergEditor = database.bool(forKey: Key.hasLaunchedGutenbergEditor)
-        return rollout.isUserIdInPhaseRolloutPercentage(account.userID.intValue) && !hasLaunchedGutenbergEditor
+        return rollout.isUserIdInPhaseRolloutPercentage(account.userID.intValue)
     }
 
     func setHasLaunchedGutenbergEditor(_ hasLaunched: Bool) {
         database.set(hasLaunched, forKey: Key.hasLaunchedGutenbergEditor)
     }
 
```

Comment:
It looks like `setHasLaunchedGutenbergEditor` isn't used anymore.

Reply:
Good catch! I will remove this unused function.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/16574/commits/26b24a7078bbed8707ab77919362febb60dffd75.